.travis.yml

```diff
@@ -13,9 +13,9 @@ before_install:
     - export PATH=/home/travis/miniconda/bin:$PATH
     - conda update --yes --quiet conda
 install:
-    - conda create -n ramp --yes pip python=$PYTHON_VERSION ipykernel
-    - source activate ramp
-    - conda env update --file environment.yml
+    - conda env create -f test_environment.yml python=$PYTHON_VERSION
+    - conda env create -f test_environment_iris_kit.yml python=$PYTHON_VERSION
+    - source activate testenv
```

Comment:
Where is the name 'testenv' coming from?

Reply:
normally this is the environment created by `environment.yml`. It seems that it named it `python=3.7` instead. I have to check why.